proposals/params-collections.md

```diff
@@ -341,6 +341,44 @@ If only a single candidate passes the test, the invocation of the candidate is s
 - the candidate is either not generic, or its type arguments are explicitly specified;
 - there is no ambiguity between normal and expanded forms of the candidate that cannot be resolved at compile time. 
 
+In this case, if the single candidate is not a local function and all the following conditions are met,
```

Comment:
For consistency/clarity the "if the single candidate is not a local function" condition could be also one of the bullet points below